docs/Configuring-Tachyon-with-Swift.md

```diff
@@ -110,10 +74,21 @@ $ ./bin/tachyon runTests
 After this succeeds, you can visit your Swift container to verify the files and directories created
 by Tachyon exist. For this test, you should see files named like:
 
-    swift://<SWIFT CONTAINER>.swift1/tachyon/data/default_tests_files/BasicFile_STORE_SYNC_PERSIST
+    swift://<SWIFT CONTAINER>/tachyon/data/default_tests_files/BasicFile_STORE_SYNC_PERSIST
 
 To stop Tachyon, you can run:
 
 ```bash
 $ ./bin/tachyon-stop.sh
 ```
+# Running functional test with IBM SoftLayer
+
+Configure your Swift or SoftLayer account in the `integration-tests/pom.xml`, where `authMethodKey` should be `keystone` or `tempauth` or `swiftauth`. 
+To run functional tests execute
+
+	mvn test -PswiftTest -pl integration-tests
```

Comment:
NIT: could we make it like
<pre>
```bash
$ mvn test -PswiftTest -pl integration-tests
```
</pre>
so the md file compiler (jekyll) gets more hint to render the html, also make it consistent with other parts of the file.